Osnova.Net/Constants/DtfSubsiteIds.cs

```diff
@@ -0,0 +1,482 @@
+﻿namespace Osnova.Net.Constants
+{
+    static class DtfSubsiteIds
+    {
+        /// <summary>
+        /// https://dtf.ru/games
+        /// </summary>
+        public const int Games = 64953;
+        /// <summary>
+        /// https://dtf.ru/gameindustry
+        /// </summary>
+        public const int GameIndustry = 87855;
+        /// <summary>
+        /// https://dtf.ru/gamedev
+        /// </summary>
+        public const int Gamedev = 64954;
+        /// <summary>
+        /// https://dtf.ru/cinema
+        /// </summary>
+        public const int Cinema = 64957;
+        /// <summary>
+        /// https://dtf.ru/life
+        /// </summary>
+        public const int Life = 128871;
+        /// <summary>
+        /// https://dtf.ru/hard
+        /// </summary>
+        public const int Hard = 94959;
+        /// <summary>
+        /// https://dtf.ru/howto
+        /// </summary>
+        public const int Gides = 112065;
+        /// <summary>
+        /// https://dtf.ru/read
+        /// </summary>
+        public const int Read = 64958;
+        /// <summary>
+        /// https://dtf.ru/sale
+        /// </summary>
+        public const int Sale = 64962;
+        /// <summary>
+        /// https://dtf.ru/indie
+        /// </summary>
+        public const int Indie = 64960;
+        /// <summary>
+        /// https://dtf.ru/avi
+        /// </summary>
```

Comment:
I thought about it lately... Subsite id's are unique and giving them our own names -- even if it's more appropriate, users will still be looking for "avi" tag, so... Let's use original tag naming AND add more appropriate name/info -- if it's needed -- in xml-docs. Sorry for inconvinience on my side.

Example of such doc:

```xml
/// <summary>
/// Refers to: https://dtf.ru/flood
/// <para/>
/// Also known as: Offtop
/// </summary>
```